packages/ebayui-core-react/src/ebay-file-preview-card-group/README.md

```diff
@@ -0,0 +1,68 @@
+# EbayFilePreviewCardGroup
+
+## Demo
+
+[Storybook](https://opensource.ebay.com/ebayui-core-react/main/?path=/docs/media-ebay-file-preview-card-group--docs)
+
+## Usage
+
+### Import JS
+
+```jsx harmony
+import { EbayFilePreviewCardGroup } from '@ebay/ui-core-react/ebay-file-preview-card-group'
+```
+
+### Import following styles from SKIN
+
+```jsx harmony
+import '@ebay/skin/file-preview-card-group'
+```
+
+### Or import styles using SCSS/CSS
+
+```jsx harmony
+import '@ebay/skin/file-preview-card-group.css'
+```
+
+### Import icons
+
+Add the below icons to the `EbaySvg` component.
+
+Note: Make sure that `EbaySvg` is only rendered on the server so it does not affect the client bundle size.
+
+```tsx
+<EbaySvg icons={['close16', 'delete16', 'vertical16', 'play16', 'file24']} />
+```
+
+```jsx harmony
+const cards = [{
```

Comment:
The way that we translate marko attribute tags is with children components. So we will use the `EbayFilePreviewCard` directly instead. 

```jsx
<EbayFilePreviewCardGroup ....>
  <EbayFilePreviewCard file={} />
  <EbayFilePreviewCard file={} />
  <EbayFilePreviewCard file={} />
</EbayFilePreviewCardGroup>
```

We use the `filterComponents()` to find the elements and render inside the `ul` and using the `as` attribute with `li`